Geometry/CaloTopology/src/HGCalTopology.cc

```diff
@@ -198,6 +198,7 @@ DetId HGCalTopology::encode(const HGCalTopology::DecodedDetId& id_) const {
 DetId HGCalTopology::changeXY(const DetId& id, int nrStepsX,
 			      int nrStepsY ) const {
 
+  /*
```

Comment:
why comment out this code instead of just deleting it?

Reply:
The newer geometry definition will use this interface later. I like to keep the interface - might delete the commented out code here.